plugins/gnostic-go-generator/main.go

```diff
@@ -36,6 +37,8 @@ func main() {
 	packageName, err := resolvePackageName(env.Request.OutputPath)
 	env.RespondAndExitIfError(err)
 
+	_, sortClientParamerList := os.LookupEnv("GNOSTIC_GO_SORTCLIENTPARAMETERLIST")
```

Comment:
I'm not sure that an environment variable is the best way to go here. `gnostic` is modeled after `protoc`, which enables plugins to take parameters/options via a plugin option, for example, if this plugin were compatible with protoc: `--go-generator-opt="sort-params"`.

Can we add this feature in a similar way? Maybe `--go-generator-sort-params` and if the `Invocation` contains this flag, we set our `renderer` toggle to `true`?